Automattic/woocommerce-payments
# Add - Tooltip and ARIA label to PM logos in Translation list and Payment Details page

Fixes #6589 

#### Changes proposed in this Pull Request

Add a tooltip indicating the brand name / payment method title to the small logo ( usually only symbol ) that appears on the Transactions list page ( Payments > Transactions ). 

The PR also adds an ARIA label for better accessibility. 

<details><summary>Desktop Screenshots</summary>

<img width="1237" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/4162931/121ec672-c438-4321-a0f2-712b64c342bb">

<img width="684" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/4162931/6b52859c-b133-44b3-979d-cac13c23c4ca">

</details> 

----

<details><summary>Mobile Screenshot - Tooltips not shown </summary>

<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/4162931/538bb55c-7310-455b-b444-5fafb49c4467">
</details> 


#### Testing instructions
* Link your local site to an account that preferably has transactions from a variety of test payment methods. 
* Browse to `Payments > Transactions` and make sure you see the `Source` column. 
* Move your mouse over the source logo, you should see a black tooltip with the name of the payment method, similar to the screenshot above. 

#### Mobile view testing
* Open the transactions page in Chrome and use the mobile view option in browser tools: 

![image](https://github.com/Automattic/woocommerce-payments/assets/4162931/f3324c6e-1ab4-49d7-8ef0-10d245ef9c1c)

* Browse to the `Payments > Transactions` page to see the transaction list. "touch" (move the mouse over, you will see a circle indicating a simulated mobile touch. click and drag that) and scroll right to see the logos column. When you touch the logo, it should not show a tooltip, rather should open up the payment details page
* On the payment details page, "touching" the logo should not bring up the tooltip. 

-------------------

- [x] Tested on mobile (or does not apply) - tested on browser simulation

**Post merge**
